app/src/main/java/com/omen273/crossLingo/SettActivity.kt

```diff
@@ -28,15 +31,29 @@ class SettActivity : AppCompatActivity() {
                 }
             }
         }
-    }
 
-    override fun onResume() {
+        move_cursor_to_filled_cell.setOnCheckedChangeListener { _, p1 ->
+            openFileOutput(MOVE_TO_FILLED_CELLS_FILE, MODE_PRIVATE).use {
+                ConfigWriter().write(it, p1)
+            }
+        }
+
         val currentLevel = ChooseTopicsActivity.readLevelFromConfig(filesDir, resources)
         if (currentLevel != null) {
             val levels = resources.getStringArray(R.array.levels)
             val index = levels.indexOf(currentLevel)
             level_list.setSelection(index)
         }
-        super.onResume()
```

Comment:
why no `onResume`?

Reply:
Because it is not onResume but onCreate so we don't need to call function from base class